### PR TITLE
new recepie for xmlbmessage cli tool to convert BMessage to XML and v…

### PR DESCRIPTION
**Pull request by Paradoxianer**
…ice versa

### CODE REVIEW

**Comment by extrowerk** [comment]
There is a type in the COPYRIGHT line.

**Comment by extrowerk** [comment]
Typo in copyright.

**Comment by extrowerk** [comment]
Revision should be 1 for new recipes

File: haiku-apps/xmlbmessage/xmlbmessage-1.0.recipe [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+REVISION="2"
+srcGitRev="68b6caab61bb78b994258c688b311334f0122168"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+
+SOURCE_FILENAME="$APP-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="$APP-$srcGitRev"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	cmd:xmlbmessage
```
Comment (korli) [inline haiku-apps/xmlbmessage/xmlbmessage-1.0.recipe]: xmlbmessage is also missing.

File: haiku-apps/xmlbmessage/xmlbmessage-1.0.recipe [hunk 2]
```diff
@@ -0,0 +1,45 @@
+SUMMARY="CLI tool to convert BMessages to XML Files"
+DESCRIPTION="A small commandline Tool for Haiku OS wich converts flattened BMessage files to XML and the other way around"
+APP="XMLBMessage"
+HOMEPAGE="https://github.com/Paradoxianer/$APP"
+COPYRIGHT="2016 - 2019 Matthias \"Paradoianer\" Lindner"
+LICENSE="MIT"
+REVISION="2"
+srcGitRev="68b6caab61bb78b994258c688b311334f0122168"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+
```
Comment (Begasus) [inline haiku-apps/xmlbmessage/xmlbmessage-1.0.recipe]: No new line here

File: haiku-apps/xmlbmessage/xmlbmessage-1.0.recipe [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+SOURCE_FILENAME="$APP-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="$APP-$srcGitRev"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```
Comment (Begasus) [inline haiku-apps/xmlbmessage/xmlbmessage-1.0.recipe]: New line between line 12 and line 13

File: haiku-apps/xmlbmessage/xmlbmessage-1.0.recipe [hunk 4]
```diff
@@ -0,0 +1,46 @@
+SUMMARY="CLI tool to convert BMessages to XML Files"
+DESCRIPTION="A small commandline Tool for Haiku OS wich converts flattened \
+			 BMessage files to XML and the other way around"
```
Comment (Begasus) [inline haiku-apps/xmlbmessage/xmlbmessage-1.0.recipe]: Suggestion:
```diff
DESCRIPTION="A small commandline Tool for Haiku OS wich converts flattened \
-			 BMessage files to XML and the other way around"
+BMessage files to XML and the other way around"
```

File: haiku-apps/xmlbmessage/xmlbmessage-1.0.recipe [hunk 5]
```diff
@@ -0,0 +1,47 @@
+SUMMARY="CLI tool to convert BMessages to XML Files"
+DESCRIPTION="A small commandline Tool for Haiku OS wich converts flattened \
+BMessage files to XML and the other way around"
+APP="XMLBMessage"
+HOMEPAGE="https://github.com/Paradoxianer/$APP"
+COPYRIGHT="2016 - 2019 Matthias \"Paradoxianer\" Lindner"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="68b6caab61bb78b994258c688b311334f0122168"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
```
Comment (Begasus) [inline haiku-apps/xmlbmessage/xmlbmessage-1.0.recipe]: Missing CHECKSUM_SHA256

File: haiku-apps/xmlbmessage/xmlbmessage-1.0.recipe [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libtinyxml
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:find
```
Comment (Begasus) [inline haiku-apps/xmlbmessage/xmlbmessage-1.0.recipe]: find comes before g++

File: haiku-apps/xmlbmessage/xmlbmessage-1.0.recipe [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+SOURCE_FILENAME="$APP-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="$APP-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	xmlbmessage = $portVersion
+	cmd:xmlbmessage
+	"
+
```
Comment (Begasus) [inline haiku-apps/xmlbmessage/xmlbmessage-1.0.recipe]: No new line between PROVIDES and REQUIRES

File: haiku-apps/xmlbmessage/xmlbmessage-1.0.recipe [hunk 8]
```diff
@@ -0,0 +1,48 @@
+SUMMARY="CLI tool to convert BMessages to XML Files"
+DESCRIPTION="A small commandline Tool for Haiku OS wich converts flattened \
```
Comment (Begasus) [inline haiku-apps/xmlbmessage/xmlbmessage-1.0.recipe]: Suggestion:
```diff
-DESCRIPTION="A small commandline Tool for Haiku OS wich converts flattened \
+DESCRIPTION="A small commandline Tool for Haiku wich converts flattened \
```

**Comment by extrowerk** [comment]
@Paradoxianer : please do not use force-push without a good reason, it makes just hard to follow, what was changed.

**Comment by Begasus** [comment]
LGTM
